### PR TITLE
Initial release of stylist UI

### PR DESCRIPTION
**Pull request by Lemongrass3110**
* **Addressed Issue(s)**: #3037

* **Server Mode**: Renewal

* **Description of Pull Request**: 
Adds support for the stylist UI.

Known TODOs:

- [x] Update to support latest kRO files
- [x] Add YAML header documentation
- [x] Add atcommand documentation
- [x] Add scriptcommand documentation

### CODE REVIEW

**Comment by lukkta2111** [comment]
adicionar definição da função openstylist em script.cpp BUILDIN_DEF(openstylist, "?"),

**Comment by RagnaWay** [comment]
No need to add the npcs hair style too? Example: ID ITEM 6959 shows the npcs that use this system.
`			"^FFFFFF<NAVI>^4D4DFF[Prontera]^000000<INFO>prt_in,243,168,0,100,0,0</INFO></NAVI>^000000",
			"^FFFFFF<NAVI>^4D4DFF[Alberta]^000000<INFO>alberta_in,58,142,0,100,0,0</INFO></NAVI>^000000",
			"^FFFFFF<NAVI>^4D4DFF[Lighthalzen]^000000<INFO>lhz_in02,100,143,0,100,0,0</INFO></NAVI>^000000",
			"^FFFFFF<NAVI>^4D4DFF[Rachel]^000000<INFO>ra_in01,186,148,0,100,0,0</INFO></NAVI>^000000",
			"^FFFFFF<NAVI>^4D4DFF[Lasagna]^000000<INFO>lasagna,134,113,0,100,0,0</INFO></NAVI>^000000",`
Video: https://www.youtube.com/watch?v=kbCa5qefWMg

**Comment by JohnnyPlayy** [comment]
Relax, I'll write the script for this function!

**Review by aleos89 — APPROVED** [review]
Final notes!

File: db/re/stylist.yml [hunk 1]
```diff
@@ -43,6 +43,12 @@ Header:
 Body:
   - Look: Hair_Color
     Options:
+      - Index: -1
```
Comment (RagnaWay) [inline db/re/stylist.yml:46]:
```
[Error]: stylist_parseBodyNode: Unsupported index value "-1"...
[Error]: Occurred in file 'db/re/stylist.yml' on line 46 and column 15.
```

**Pull request merged**